docs/status.md

```diff
@@ -27,7 +27,7 @@ import MoreHelp from "/src/components/MoreHelp"
 | Ruby       |                            | Rust                       |
 | TypeScript |                            | YAML                       |
 | TSX        |                            | Generic (ERB, Jinja, etc.) |
-|            |                            |                            |
+|            |                            |  Solidity                          |
```

Comment:
Could you place this below Rust and above YAML so they remain in alphabetical order?

Reply:
@pabloest Yup, working on it now